app/routes/goal_routes.py

```diff
@@ -0,0 +1,103 @@
+from app import db 
+from app.models.goal import Goal 
+from app.models.task import Task
+from flask import Blueprint, make_response, request, jsonify, abort
+
+goal_bp = Blueprint("goal", __name__, url_prefix="/goals")
+
+@goal_bp.route("", methods=["POST"])
+def create_goal(): 
+    request_body = request.get_json()
+
+    if "title" not in request_body: 
+        return make_response({"details" : "Invalid data"}, 400)
```

Comment:
Note that the `make_response()` function isn't necessary here. `make_response()` should be used when we need access to the Response Object to update the headers or in situations where we need to pass the Response Object to another function like `abort()`. Otherwise, flask will handle converting our returned dictionary and status code into a proper HTTP Response Object. Yay Flask!